src/helpers/user.helpers.js

```diff
@@ -0,0 +1,25 @@
+const userService = require('../services/user.service');
+const nameCheck = (name) => {
+  if (!name || typeof name !== 'string') {
+    return true;
+  }
```

Comment:
```suggestion
  return !name || typeof name !== 'string';
```

Will it work the same?